modules/logging-release-notes-5-9-10.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging_release_notes/logging-5-9-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-10_{context}"]
+= Logging 5.9.10
+
+This release includes link:https://access.redhat.com/errata/RHSA-2024:10990[{logging-uc} {for} Bug Fix Release 5.9.10].
```

Comment:
I think so far we have only repeated the RHSA name in the link:

```suggestion
This release includes link:https://access.redhat.com/errata/RHSA-2024:10990[RHSA-2024:10990].
```

Reply:
Hm. I have looked at a few different release notes and this seems to be something that we have done for a few recent releases only. I'm not sure which way I prefer. We can also keep this as it currently is in the PR.

Reply:
Thank you for spotting this. Yes, it's currently inconsistent across releases. I've updated it to match the other headings on the same page, so it will be consistent within each page. Thank you again.